app/external_messaging/telegram.py

```diff
@@ -1,3 +1,18 @@
-# TODO: implement a concrete implementation in the future
-def post_telegram_message(account: str, msg: str):
-    print(f"[TelegramBot - {account}]: {msg}")
+# TODO: sample Python code that sends a text message to a user on Telegram using the python-telegram-bot library
+
+'''
+Make sure to replace YOUR_BOT_TOKEN_HERE with your bot token 
+and USER_CHAT_ID_HERE with the chat ID of the user you want to send the message to.
+You can obtain a bot token by creating a new bot with BotFather on Telegram.
+'''
+
+# function send_message which takes in two arguments: text and chat_id, 
+# sends the text message to the specified user on Telegram
+from telegram import Bot
+
+def send_message(text: str, chat_id: str):
```

Comment:
As you can see on [this line](https://github.com/dominik-air/lol-afk-buddy/blob/b9e76336803922bd5f60dac33ec34f471eea3422/app/events/telegram_listener.py#L2) we are importing the post_telegram_message function from this file. We have to somehow normalise the naming conventions. You can either change every `post_telegram_message` function to `send_message` in the [telegram_listener.py](https://github.com/dominik-air/lol-afk-buddy/blob/b9e76336803922bd5f60dac33ec34f471eea3422/app/events/telegram_listener.py) file or change `send_message` function name back to `post_telegram_message`. Also remember to change the parameters' names. I was using `account` and `msg`, but you are using `chat_id` and `text` respectively.

Reply:
bello cool @dominik-air, 
Thanks for your message and for explaining the necessary changes to the code. I completely understand the need for consistent naming conventions, and I'll make sure to update the function names and parameter names accordingly.

I really appreciate the opportunity to contribute to the project, and I'm glad that the observer pattern has been implemented to make adding messaging app support easier. I'll work on making the changes you suggested and let you know as soon as they're complete.

![meow](https://media.giphy.com/media/xHApYE9nEuu2s/giphy.gif)